skills/wipe_online.py

```diff
@@ -0,0 +1,696 @@
+import argparse
```

Comment:
can we combine this with the previous file? two separate files seems a bit messy/redundant